format/proto/types.proto

```diff
@@ -27,3 +27,10 @@ message Metadata {
     optional sfixed32 checksum = 1;
     optional sfixed32 length = 2;
 }
+
+message NodeMetrics {
+    optional string endpoint = 1;
+    optional bool layoutServerStatus = 2;
```

Comment:
Yes, I have a few more in mind. But at this time I am not sure of the format.
So I will add them in subsequent patches if that's fine.